internal/scripts/docker_env/scripts/xesmf_env.sh

```diff
@@ -25,5 +25,9 @@ ENV_NAME=xesmf.${METPLUS_VERSION}
 # Conda environment to use as base for new environment
 BASE_ENV=metplus_base.${METPLUS_VERSION}
 
-mamba create -y --clone ${BASE_ENV} --name ${ENV_NAME}
-mamba install -y --name ${ENV_NAME} -c conda-forge netcdf4==1.7.2 xarray==2025.1.2 xesmf esmf
+conda create -y --clone ${BASE_ENV} --name ${ENV_NAME}
+mamba install -y --name ${ENV_NAME} -c conda-forge \
```

Comment:
line 29: do you also want to replace this from mamba to conda?

Reply:
Oddly, in a newer version of mamba that is on seneca (2.0.5), mamba does not have the `--clone` option, so this step fails. I changed these create-from-clone commands to `conda` from `mamba` so they would work on seneca. I'm not sure why the option was removed from mamba?

Reply:
The subsequent mamba commands to install the packages works even if you created the env via conda and runs much faster, so I left those to use mamba